docs/specs/shapes.md

```diff
@@ -7,6 +7,52 @@ The graphical elements are divided in 4 categories:
 * [Styles](#shape-style), that define the visual appearance of shapes
 * [Modifiers](#modifier) alter the curves of the shapes
 
+## Shape Rendering Model
+
+### Grouping and Ordering Rules
+
+* **shapes** are rendered in reverse order (bottom->top)
```

Comment:
Possibly implied already but should we also call out this is depth first? Practically speaking, this means shapes at the top of the file will also be on top of the view?